git.go

```diff
@@ -385,7 +385,7 @@ func (s *GitRepo) ExportDir(dir string) error {
 		return NewLocalError("Unable to export source", err, string(out))
 	}
 	// and now, the horror of submodules
-	out, err = s.RunFromDir("git", "submodule", "foreach", "--recursive", "'git checkout-index -f -a --prefix=\""+filepath.Join(dir, "$path")+"\"'")
+	out, err = s.RunFromDir("git", "submodule", "foreach", "--recursive", "git checkout-index -f -a --prefix=\""+filepath.Join(dir, "$path")+"/\"")
```

Comment:
By the way, is the newly added `/` (forward slash) character at the intentional?

I ask because the commit message only mentions removing single quotes, nothing about adding a slash.